src/utils/common.ts

```diff
@@ -53,3 +55,24 @@ export const addressesEqual = (address1: string | undefined | null, address2: st
 
   return isCaseInsensitiveMatch(address1, address2);
 };
+
+export const switchWalletProviderToChain = async (chainId: number): Promise<boolean> => {
+  // @ts-ignore
+  if (!window?.ethereum) {
+    alert('Unsupported browser!');
```

Comment:
Just wondering if an alert is  too much / needed here as opposed to a console.error or throw?

Reply:
now that I read I think you're right, since it's a toolkit we should leave all the UI up to dev, I will remove alerts and replace with console warns + error message returned as result

Reply:
thanks, fixed